src/Symfony/Component/HttpKernel/composer.json

```diff
@@ -20,7 +20,7 @@
         "symfony/error-handler": "^4.4",
         "symfony/event-dispatcher": "^4.4",
         "symfony/http-client-contracts": "^1.1|^2",
-        "symfony/http-foundation": "^4.4|^5.0",
+        "symfony/http-foundation": "^4.4.21|^5.2.6",
```

Comment:
should be bumped to `^4.4.22|^5.2.7` now that 4.4.21 and5.2.6 have been released